src/Abstractions/Games/CardsPoint.cs

```diff
@@ -0,0 +1,286 @@
+
+namespace BoardGames.Abstractions.Games
+{
+    public enum CardSuit
+    {
+        Diamonds,
+        Hearts,
+        Spades,
+        Clubs
+    }
+
+    public enum CardRank
+    {
+        Jack = 3,
+        Queen = 4,
+        King = 5,
+        Six = 6,
+        Seven = 7,
+        Eight = 8,
+        Nine = 9,
+        Ten = 10,
+        Ace = 11
+    }
+
+    public enum Status
+    {
+        Active,
+        Finished
+    }
+
+    public record Card
+    {
+        public int Id { get; set; }
+        public CardSuit Suit { get; set; }
+        public CardRank Rank { get; set; }
+
+        public Card(int id, CardSuit suit, CardRank rank)
+        {
+            Id = id;
+            Suit = suit;
+            Rank = rank;
+        }
+    }
+
+    public record PointState(ImmutableList<Card> Cards,
+        ImmutableList<int> TotalScores,
+        int MoveIndex = 0,
+        int FirstPlayerIndex = 0,
+        int PlayersCount = 0)
+    {
+        public ImmutableList<ImmutableList<Card>> PlayersCards { get; set; } = ImmutableList<ImmutableList<Card>>.Empty;
+        public ImmutableList<Status> Statuses { get; set; } = ImmutableList<Status>.Empty;
+        public ImmutableList<int> Scores { get; set; } = ImmutableList<int>.Empty;
+        public ImmutableList<int> TotalScores { get; set; } = ImmutableList<int>.Empty;
+        public int PlayerIndex => (MoveIndex + FirstPlayerIndex) % PlayersCount;
+
+        public PointState() : this((ImmutableList<Card>)null!, (ImmutableList<int>)null!) { }
+
+        public PointState(int playersCount) : this(ImmutableList<Card>.Empty, ImmutableList<int>.Empty)
+        {
+            var cards = new List<Card>();
+            var cardId = 1;
+            foreach (var suit in (CardSuit[])Enum.GetValues(typeof(CardSuit))) {
+                foreach (var rank in (CardRank[])Enum.GetValues(typeof(CardRank))) {
+                    var card = new Card(cardId, suit, rank);
+                    cards.Add(card);
+                    cardId++;
+                }
+            }
+
+            Cards = cards.ToImmutableList();
+            var statuses = new List<Status>();
+            var playersCards = new List<ImmutableList<Card>>();
+            var scores = new List<int>();
+            for (var i = 0; i < playersCount; i++) {
+                statuses.Add(Status.Active);
+                playersCards.Add(ImmutableList<Card>.Empty);
+                scores.Add(0);
+            }
+
+            TotalScores = Scores = scores.ToImmutableList();
+            Statuses = statuses.ToImmutableList();
+            PlayersCards = playersCards.ToImmutableList();
+            MoveIndex = 0;
+            FirstPlayerIndex = 0;
+            PlayersCount = playersCount;
+        }
+    }
+
+    public record PointMove(bool IsSkip) : GameMove
+    { }
+
+    [Service, ServiceAlias(typeof(IGameEngine), IsEnumerable = true)]
+    public class PointEngine : GameEngine<PointState, PointMove>
```

Comment:
I'd rename this to PointGameEngine, PointGameState, and PointGameMove - PointXxx is a bit confusing.